### PR TITLE
task done)

### PR DESCRIPTION
**Pull request by AndrewOliferchuk**

### CODE REVIEW

File: src/main/java/com/example/demo/model/Book.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+public class Book {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private String title;
+    private String author;
+    private String isbn;
+    private BigDecimal price;
+    private String description;
+    private String coverImage;
```
Comment (MAMentorFX) [inline src/main/java/com/example/demo/model/Book.java]: image

Comment (MAMentorFX) [reply src/main/java/com/example/demo/model/Book.java]: Add validation annotations

**Review by MAMentorFX — CHANGES_REQUESTED** [review]

File: src/main/java/com/example/demo/model/Book.java (hunk 1 above)
Comment (AndrewOliferchuk) [reply src/main/java/com/example/demo/model/Book.java]: @Column(nullable = false, unique = true)
private String isbn;
типу якось так має бути?

File: src/main/java/com/example/demo/model/Book.java [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import java.math.BigDecimal;
+import lombok.Data;
+
+@Entity
+@Data
```
Comment (koval-kn) [inline src/main/java/com/example/demo/model/Book.java]: dont use data with entites as it cause bugs, replace with getter/settet annotations
https://stackoverflow.com/questions/75181366/why-jpa-buddy-complains-about-data-annotation-over-jpa-entity

**Review by koval-kn — CHANGES_REQUESTED** [review]
almost perfect

**Review by Vasul-Pankiv — APPROVED** [review]

File: src/main/java/com/example/demo/dto/CreateBookRequestDto.java [hunk 3]
```diff
@@ -0,0 +1,14 @@
+package com.example.demo.dto;
+
+import java.math.BigDecimal;
+import lombok.Data;
+
+@Data
+public class CreateBookRequestDto {
+    private String title;
```
Comment (Elena-Bruyako) [inline src/main/java/com/example/demo/dto/CreateBookRequestDto.java]:
```suggestion
    @NotBlank
    private String title;
```

Comment (Elena-Bruyako) [reply src/main/java/com/example/demo/dto/CreateBookRequestDto.java]: Add validation for all necessary fields

File: src/main/java/com/example/demo/model/Book.java [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+@Table(name = "books")
+public class Book {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @Column(nullable = false)
+    private String title;
+    @Column(nullable = false)
+    private String author;
+    private String isbn;
```
Comment (Elena-Bruyako) [inline src/main/java/com/example/demo/model/Book.java:23]:
```suggestion
    @Column(unique = true, nullable = false)
    private String isbn;
```

**Review by Elena-Bruyako — CHANGES_REQUESTED** [review]
Good job! Let's improve your solution!

File: pom.xml [hunk 5]
```diff
@@ -20,14 +20,60 @@
...
+		<dependency>
+			<groupId>org.springframework</groupId>
+			<artifactId>spring-web</artifactId>
+			<version>6.1.1</version>
+		</dependency>
+		<dependency>
+			<groupId>org.apache.ant</groupId>
+			<artifactId>ant</artifactId>
+			<version>1.8.1</version>
+		</dependency>
```
Comment (Rommelua) [inline pom.xml]: remove these dependencies - use spring-boot starters

File: src/main/java/com/example/demo/dto/CreateBookRequestDto.java (hunk 3 above)
Comment (Rommelua) [reply src/main/java/com/example/demo/dto/CreateBookRequestDto.java]: Column - is for entity classes
We do not need validation for now. We will have a separate task to add it.

File: src/main/java/com/example/demo/model/Book.java [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+import jakarta.persistence.Table;
+import java.math.BigDecimal;
+import lombok.Getter;
+import lombok.Setter;
+
+@Entity
+@Getter
+@Setter
+@Table(name = "books")
+public class Book {
```
Comment (Rommelua) [inline src/main/java/com/example/demo/model/Book.java:15]: Why do we have this class in this PR?

**Review by Rommelua — CHANGES_REQUESTED** [review]

File: src/main/java/com/example/demo/model/Book.java (hunk 6 above)
Comment (AndrewOliferchuk) [reply src/main/java/com/example/demo/model/Book.java:15]: А чому не має?

**Pull request merged**